src/renderer/components/status-bar/status-bar-registration.ts

```diff
@@ -23,6 +25,11 @@ export interface StatusBarComponents {
    * @default "right"
    */
   position?: "left" | "right";
+
+  /**
+   * If specified, controls item visibility
+   */
+  visible?: IComputedValue<boolean>;
```

Comment:
Is this actually the best place for the `visible` field? Should it be a field in `StatusBarRegistration`, to be consistent with the other registrations (like `ClusterPageMenuRegistration`, `KubeObjectMenuRegistration`, etc)

Reply:
Good point

Reply:
I'll change it

Reply:
Changed, please take a look.